substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -148,11 +166,11 @@ pub mod pallet {
 		traits::{tokens::GetSalary, EnsureOrigin},
 	};
 	use frame_system::{ensure_root, pallet_prelude::*};
-
-	/// Number of available ranks.
-	pub(crate) const RANK_COUNT: usize = 9;
+	/// The in-code storage version.
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Please also deploy the migration to the westend-collectives chain.

Reply:
done